viewer/packages/cad-parsers/src/cad/filterPrimitivesCommon.ts

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import * as THREE from 'three';
+
+export function filterPrimitivesOutsideClipBox(
+  attributesByteValues: Uint8Array,
```

Comment:
Maybe just pass in `ArrayBuffer` instead? And create `Uint8Array` and `Float32Array` views within the function as required?